BLSQ/iaso
# IA-1465 / IA-2180: Planning allow parent picking on list view

Parent picking mode should work in the list of org units in microplanning.

Related JIRA tickets : IA-1465 IA-2180

## Self proofreading checklist


## Changes

- Move diaolg to index
- allow list to set selected parent and open the dialog
- Typing refactor
- used redirec to replace while changing filters (IA-2180)

## How to test

Find a working planning with teams, org unit with shapes and if you can already assigned org units
Select the list tab, enable the parent picking mode and click on a row, you should see the dialog allowing to assign the team / user to the children org units

## Print screen / video


https://github.com/BLSQ/iaso/assets/12494624/e0faf8ff-bf68-4df4-ab15-855b821a90e9
